etc/profile-a-l/git-cola.profile

```diff
@@ -62,5 +62,5 @@ dbus-user filter
 dbus-system none
 
 read-only ${HOME}/.ssh
-read-only ${HOME}/.gnupg
+# read-only ${HOME}/.gnupg
```

Comment:
why was this necessary?

Reply:
`gpg` creates a temporary file under `.gnupg` https://imgur.com/N1n5Jpt, I forgot when I made the profile and cola passes this to git: git -c diff.suppressBlankEmpty=false -c log.showSignature=false commit -F git-cola-commit-message-lk887bkx -v --gpg-sign`. I can uncomment and put a comment above like the `private-etc` one.